webapp/src/ts/components/filters/analytics-filter/analytics-filter.component.ts

```diff
@@ -30,9 +65,67 @@ export class AnalyticsFilterComponent implements AfterContentInit, AfterContentC
     this.subscriptions.unsubscribe();
   }
 
+  private subscribeToStore() {
+    const subscription = this.store.select(Selectors.getSidebarFilter).subscribe(({ isOpen }) => {
+      this.isOpen = isOpen;
+    });
+    this.subscriptions.add(subscription);
+  }
```

Comment:
```suggestion
  private subscribeToStore() {
    const subscription = this.store
      .select(Selectors.getSidebarFilter)
      .subscribe(({ isOpen }) => this.isOpen = isOpen);
    this.subscriptions.add(subscription);
  }
```
On function chaining, each function goes in separate lines